### ISSUE TITLE
<rejected> Error: marked(): input parameter is of type [object Array], string expected

### ISSUE THREAD

**Issue by TeJay97** [compodoc/compodoc#1123]
I tried adding Compodoc to an existing Angular Project, but getting this error:
```
Unhandled Rejection at: Promise {
  <rejected> Error: marked(): input parameter is of type [object Array], string expected
      at marked (C:\Users\<USER>\projects\<myProject>\node_modules\marked\src\marked.js:27:11)
      at C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:600:23
      at arrayEach (C:\Users\<USER>\projects\<myProject>\node_modules\lodash\lodash.js:530:11)
      at Object.forEach (C:\Users\<USER>\projects\<myProject>\node_modules\lodash\lodash.js:9410:14)       
      at markedtags (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:599:18)
      at ClassHelper.visitClassDeclaration (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:6661:33)
      at C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:9711:63
      at Array.reduce (<anonymous>)
      at AngularDependencies.getClassIO (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:9708:33)
      at AngularDependencies.processClass (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:8679:23)
} reason: Error: marked(): input parameter is of type [object Array], string expected
    at marked (C:\Users\<USER>\projects\<myProject>\node_modules\marked\src\marked.js:27:11)
    at C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:600:23
    at arrayEach (C:\Users\<USER>\projects\<myProject>\node_modules\lodash\lodash.js:530:11)
    at Object.forEach (C:\Users\<USER>\projects\<myProject>\node_modules\lodash\lodash.js:9410:14)
    at markedtags (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:599:18)
    at ClassHelper.visitClassDeclaration (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:6661:33)
    at C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:9711:63
    at Array.reduce (<anonymous>)
    at AngularDependencies.getClassIO (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:9708:33)
    at AngularDependencies.processClass (C:\Users\<USER>\projects\<myProject>\node_modules\@compodoc\compodoc\dist\index-cli-32a621dc.js:8679:23)
```

OS: Windows 10
node: v14.17.5
npm: 6.14.14
Angular CLI: 11.2.14
typescript: 4.1.6

Installed Compodoc with ```npm i -g compodoc```

I am not entirely sure what to look for.

**Comment by vogloblinsky** [comment]
Can i have the sourcecode of the file where it breaks ?

**Comment by kazwtnb** [comment]
I had the same problem. And I use lodash too.

>   at arrayEach (C:\Users\<USER>\projects\<myProject>\node_modules\lodash\lodash.js:530:11)

The above may be the cause of this problem.

here's 'arrayEach' function in lodash.js

```
  /**
   * A specialized version of `_.forEach` for arrays without support for
   * iteratee shorthands.
   *
   * @private
   * @param {Array} [array] The array to iterate over.
   * @param {Function} iteratee The function invoked per iteration.
   * @returns {Array} Returns `array`.
   */
  function arrayEach(array, iteratee) {
    var index = -1,
        length = array == null ? 0 : array.length;

    while (++index < length) {
      if (iteratee(array[index], index, array) === false) {  // Line 530 is here

        break;
      }
    }
    return array;
  }
```

**Comment by vogloblinsky** [comment]
Fixed with last commits, coming in 1.1.15
